org.natron.Natron.json

```diff
@@ -0,0 +1,856 @@
+{
+    "id": "org.natron.Natron",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "Natron",
+    "rename-desktop-file": "natron.desktop",
+    "rename-appdata-file": "natron.xml",
+    "rename-icon": "natron",
+    "finish-args": [
+            "--share=ipc",
+            "--share=network",
```

Comment:
Does it need network access?

Reply:
Yes, for Documentation, Forum, etc links

Reply:
As in, user can access these from inside the application?

Reply:
If you're just opening links in a web browser then the app itself doesn't need network access, if you're showing things in an embedded browser then you do.